.github/workflows/depandabot.yml

```diff
@@ -0,0 +1,36 @@
+name: 'Dependabot Automerge - Action'
+on:
+  pull_request:
+jobs:
```

Comment:
Can you rename to `dependabot-automerge.yml` (so it's not confused with `dependabot.yml`)?